SWLOR.Game.Server/Service/CombatPoint.cs

```diff
@@ -168,14 +140,12 @@ static int CalculateAdjustedXP(int highestRank, int baseXP, SkillType skillType,
                     }
 
                     // Armor XP is calculated the same way but is separate from other skills used during combat.
-                    var armorPoints = CalculateArmorPoints(player);
-                    if (armorPoints <= 0) return;
                     var armorRank = dbPlayer.Skills[SkillType.Armor].Rank;
 
                     delta = npcLevel - armorRank;
                     baseXP = Skill.GetDeltaXP(delta);
-                    var xp = CalculateAdjustedXP(armorRank, baseXP, SkillType.Armor, totalPoints, armorPoints, dbPlayer.Skills);
-                    Skill.GiveSkillXP(player, SkillType.Armor, xp);
+                    var xp = baseXP * CalculateRankRangePenalty(highestRank, armorRank);
```

Comment:
I'm thinking maybe we should base the penalty off the NPC level vs the player's armor level instead of tying it back to the weapon. In legacy, we tie it by the highest skill used but I kind of think we should change that. Thoughts?

Reply:
baseXP is calculated from the difference between the NPC rank and the armour level, so NPC level is factored in. 

I think you put the rank range penalty in to avoid high level characters levelling lowbies by dragging them around with them?  I'm just trying to preserve that function here, but I'm very happy to remove it.

Reply:
I think that was the original intention but I'm far less concerned about it now. Let's go ahead and remove it unless you feel strongly about keeping it.